redhat/varnish.spec

```diff
@@ -127,6 +127,7 @@ rm -rf %{buildroot}
 %{_unitdir}/*
 %exclude %{_datadir}/%{name}/vmodtool*
 %exclude %{_datadir}/%{name}/vsctool*
+README*
```

Comment:
Are you sure about this?

I think it's `%doc README*`

Reply:
You are correct, thanks.